### ISSUE TITLE
Detector rails do not generate power

### ISSUE THREAD

**Issue by Woazboat** [cuberite/cuberite#3051]
Detector rails are not generating any power when active.

**Comment by SphinxC0re** [comment]
Note: Detector rails do only propagate power directly connected powered rails

**Comment by Woazboat** [comment]
Do you mean at the moment in Cuberite or in general? They aren't doing anything for me when activated, except mess up and reorient adjacent rails. In vanilla they can power any adjacent blocks.

**Comment by SphinxC0re** [comment]
I mean, they are doing it in Vanilla

**Comment by Woazboat** [comment]
Nope: http://minecraft.gamepedia.com/Detector_Rail#Redstone_component

**Comment by SphinxC0re** [comment]
Oh, I'm sorry, I misread the title. Powered rails propagate a redstone signal only to adjacent powered rails.
